crates/smartmodule-development-kit/tests/cmd.rs

```diff
@@ -0,0 +1,32 @@
+use std::process::Command;
+
+#[test]
+fn test_error_is_redirect_to_stderr() {
+    let cargo_manifest_dir = env!("CARGO_MANIFEST_DIR");
+    let (_, stdout) = handle_cmd_outputs(Command::new("cargo").args(&[
+        "run",
+        "--manifest-path",
+        &format!("{}/Cargo.toml", cargo_manifest_dir),
+        "--",
+        "test",
+        "--text",
+        "foo",
+        "--wasm-file",
+        &format!("{}/fixtures/filter_error.wasm", cargo_manifest_dir),
+    ]));
```

Comment:
It is possible to mimic the `fluvio-smartengine` tests using `#[ignore]` along side `make` dependencies but I think it is easier to just call `cargo` directly.

https://github.com/infinyon/fluvio/blob/ac4453b51320713700fcbedee5d1d469575fac36/makefiles/check.mk#L45